packages/amplify-category-geo/src/provider-controllers/import.ts

```diff
@@ -113,4 +119,26 @@ const bulkUploadGeofence = async (params: GeofenceCollectionParams, region: stri
     throw new Error(`The uploaded geofences should have at most ${MAX_ENTRIES_PER_BATCH} items per batch.`);
   }
   return await service.batchPutGeofence(params).promise();
-}
\ No newline at end of file
+};
+
+/**
+ * Scan the custom properties as candidate options
+ * @param geoJSONObj Geo features collection
+ * @returns Array of candidate custom property names
+ */
+const scanCandidateCustomProperties = (geoJSONObj: FeatureCollection): string[] => {
+  const { features } = geoJSONObj;
+  if (features.length > 0) {
+    const candidateProperties: string[] = Object.keys(features[0].properties).filter(prop => {
```

Comment:
It is not required to have a `properties` field on a Feature. We have to handle cases when it is not present on any Feature inside the collection. I've verified this using Location console.